.env.example

```diff
@@ -0,0 +1,2 @@
+BLIZZARD_CLIENT_ID= # Create a client and paste a ClientID. Instruction: https://develop.battle.net/documentation/guides/getting-started
```

Comment:
So, creation of api keys is not very stable. I try 3 days in a row, before creation page becomes working.